src/constants/MarkdownFiles/posts/2025-06-22-DMP-25-justin212407-week3.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "DMP’25 Week 02 Update by Justin Charles"
```

Comment:
Title says "Week 02"

Did you try your code, because the errors are obvious when you test it out. For example:

![Screenshot from 2025-07-01 10-26-49](https://github.com/user-attachments/assets/ee130def-f7e5-4113-a1df-200377bca6c9)